ruoyi-common/src/test/java/com/ruoyi/common/support/ConvertTest.java

```diff
@@ -0,0 +1,203 @@
+package com.ruoyi.common.support;
+
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.text.ParseException;
+import java.util.Set;
+import java.util.TreeSet;
+
+import static com.ruoyi.common.support.Convert.*;
+import static org.junit.Assert.*;
```

Comment:
Please don't use wildcarding for imports for `org.junit.Assert.*;`, import each one individually.

Reply:
Done